lib/discordrb/data/server.rb

```diff
@@ -585,9 +585,13 @@ def max_emoji
       end
     end
 
+    # Retrieve banned users from this server.
+    # @param limit [Integer] Number of users to return (up to maximum 1000, default 1000).
+    # @param before_id [Integer] Consider only users before given user id.
+    # @param after_id [Integer] Consider only users after given user id.
     # @return [Array<ServerBan>] a list of banned users on this server and the reason they were banned.
-    def bans
-      response = JSON.parse(API::Server.bans(@bot.token, @id))
+    def bans(limit = nil, before_id = nil, after_id = nil)
```

Comment:
Would prefer to see these as kwargs

Reply:
Should I assume that the whole chain of PRs is concerned to be accepted?
Just to do like for the emoji test and add it as a commit at the end of the chain on this branch, or do you want me to do it properly in the concerned branch and rebase and force push all the following ones?

Reply:
We can work off this PR and merge it all at once

Reply:
Ok I'll do that then!